canonical/ubuntu.com
# Footer

## Done

I hacked this to work... god knows how many dead selectors I left in my wake... needs a serious re-factor
## QA
1. go to any footer
2. look on small/med/large
3. compare to live
4. see that Things is now IoT
5. file issues with @grahambancroft 
## Issue / Card

Fixes #273
Fixes #36 
Fixes #37 
Fixes #38
Fixes https://github.com/ubuntudesign/ubuntu-vanilla-theme/issues/38
## Screenshots

Small

![image](https://cloud.githubusercontent.com/assets/441217/14412641/d074c9c4-ff5e-11e5-92b3-16f4b36333a3.png)

Medium

![image](https://cloud.githubusercontent.com/assets/441217/14412700/41f56c88-ff60-11e5-9140-43025a47385a.png)

Large

![image](https://cloud.githubusercontent.com/assets/441217/14412637/b1d418b2-ff5e-11e5-8331-4dbc1f965f60.png)
